packages/services/emails/src/mjml.ts

```
@@ -0,0 +1,101 @@
+ * @source https://github.com/component/escape-html
+ */
+
+function escapeHtml(input: string): string {
+  const matchHtmlRegExp = /["'<>]/;
+  const match = matchHtmlRegExp.exec(input);
+
+  if (!match) {
+    return input;
+  }
+
+  let escape;
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)